README.md

```diff
@@ -160,9 +157,6 @@ framework:
             my_sns: 
                 dsn: 'sns://arn:aws:sns:us-east-1:1234567890:foobar'
 
-bref_messenger:
-    sns: true # Register the SNS transport
-
 services:
     Aws\Sns\SnsClient:
```

Comment:
There must be a service called `Aws\Sns\SnsClient` when you are using a DSN `sns://...`. With this PR you may now configure that with `bref_messenger.transports.sns.client: 'foobar_service'`

Reply:
In which cases can this be helpful?

Reply:
When you have multiple SNS clients or if you have existing services that you don't want to rename.